pkg/registry/csv.go

```diff
@@ -341,3 +352,48 @@ func (csv *ClusterServiceVersion) GetOperatorImages() (map[string]struct{}, erro
 
 	return images, nil
 }
+
+type Icon struct {
+	MediaType  string `json:"mediatype"`
+	Base64data string `json:"base64data"`
```

Comment:
If you use `Base64data []byte`, the json unmarshaller will automatically decode the base64 string into the binary []byte data.

Since this is a new type and it is presumably only being used in the new `opm add` command, I think we could start enforcing that any newly added bundles must have valid icon data, and error out if they don't.

The only catch is that if index maintainers want to rebuild existing indexes in this format from existing bundles that have this issue, that won't work. In that case, perhaps we could log a warning and skip setting the icon?